engine/link/mystery_gift.asm

```diff
@@ -1186,15 +1186,15 @@ ReceiveIRDataMessage:
 	inc d
 	jr z, .recv_done
 	ldh a, [c]
-	bit rRP_RECEIVING, a
+	bit RPF_WRITE_HI, a
```

Comment:
```suggestion
	bit 1, a
```
This matches `RPF_DATAIN`. Maybe `RPB_DATAIN` should be added?

Reply:
Might be worth an addition for those RPB constants yes

Reply:
Added a comment explaining such for now

Reply:
Making this change breaks the build and causes a nonmatching, undoing and keeping the comment